src/qt/emuwindow.cpp

```diff
@@ -75,9 +76,16 @@ int EmuWindow::init(int argc, char** argv)
 
     char* bios_name = nullptr, *file_name = nullptr, *gsdump = nullptr;
 
+    // Load before the arguments, so the arguments override the config.
+    Settings::load();
+    if (!Settings::bios_path.isEmpty()) bios_name = Settings::as_char(Settings::bios_path);
+
     ARGBEGIN {
         case 'b':
             bios_name = ARGF();
+
+            // Save the bios path in settings, converting to a QString along the way.
+            Settings::bios_path = QString::fromLocal8Bit(bios_name);
```

Comment:
Perhaps we should only save the BIOS path if the file is good?

Reply:
That may be a misleading comment, I suppose. It stores the bios path in settings, but doesn't write the settings at that point. It doesn't get written to a file until we get to Settings::save(); later, and you don't get that far if the file wasn't good.